### PR TITLE
Add join message and arrow indicators

### PR DESCRIPTION
**Pull request by TwoLeggedCat**

### CODE REVIEW

File: src/main/java/github/granny/quill/modules/JoinLeaveLogger.java [hunk 1]
```diff
@@ -3,12 +3,14 @@
...
 import github.granny.quill.util.TimeUtil;
 import net.dv8tion.jda.api.entities.TextChannel;
+import net.dv8tion.jda.api.events.guild.member.GuildMemberJoinEvent;
 import net.dv8tion.jda.api.events.guild.member.GuildMemberRemoveEvent;
 import net.dv8tion.jda.api.hooks.ListenerAdapter;
 
 import javax.annotation.Nonnull;
 import java.time.OffsetDateTime;
 import java.time.temporal.ChronoUnit;
+import java.util.concurrent.TimeUnit;
```
Comment (notOM3GA) [inline src/main/java/github/granny/quill/modules/JoinLeaveLogger.java:13]: unneeded import

**Review by notOM3GA — CHANGES_REQUESTED** [review]

**Review by notOM3GA — APPROVED** [review]
tested, lgtm

**Comment by YouHaveTrouble** [comment]
Maybe make it a small embed instead? Not sure if I like just a plain message for this

**Comment by granny** [comment]
> Maybe make it a small embed instead? Not sure if I like just a plain message for this

I'd rather not. Embeds are too bulky. The effect we're trying to replicate is discord's default welcome message.

**Comment by TwoLeggedCat** [comment]
> Maybe make it a small embed instead? Not sure if I like just a plain message for this

Made it to match the original join message, and I kinda like the plain message because it takes like no space (Syscraft's huge messages always annoy me). When having a conversation in general, I don't really want a giant blob appearing in the middle of it.

**Comment by TwoLeggedCat** [comment]
I can play around with it if you want, but the compactness seems like a plus to me.

**Review by granny — APPROVED** [review]

**Comment by granny** [comment]
> I can play around with it if you want, but the compactness seems like a plus to me.

Don't worry about it. I'm working on another feature right now so after that's done I'll merge this.

**Pull request merged**